samples/matter/template/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay

```diff
@@ -49,3 +49,7 @@
 &uart30 {
 	status = "okay";
 };
+
+&wdt30 {
```

Comment:
So this is already enabled for other boards, right?

Reply:
Yes, only for nRF54L15 it is required to do it manually for now

Reply:
This is done in the same way in Zephyr's sample